src/controllers/userController.ts

```diff
@@ -0,0 +1,89 @@
+import express, { Router, Request, Response } from "express";
+import User from "../models/user";
+import crypto from "crypto";
+import jwt from 'jsonwebtoken';
+
+class UserController {
+  static key: any = process.env.key;
+  static iv: any = process.env.iv;
+  static algo: any = process.env.algo;
+  public static register = (req: Request, res: Response) => {
+    res.render("user/register");
+  };
+
+  public static signin = (req: Request, res: Response) => {
+    res.render("user/signin");
+  };
+
+  public static registerUser = async (req: Request, res: Response) => {
+    const body = req.body;
+    const { email, pwd } = req.body;
+    const password = this.encryptData(pwd);
+    const user = new User({
+      email,
+      password,
+    });
+    console.log(user);
+    try {
+      const newUser = await user.save();
+      console.log(`user added successfully ${newUser}`);
+      res.render("user/signin");
+    } catch (error) {}
```

Comment:
Don't leave catch blocks empty, you should handle the error or add logs to understand why its failing

Reply:
done